jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.editor
+
+import com.intellij.util.text.DateFormatUtil
+import com.intellij.util.ui.ColumnInfo
+import com.intellij.util.ui.ListTableModel
+import software.amazon.awssdk.services.cloudwatchlogs.model.LogStream
+import software.aws.toolkits.resources.message
+import javax.swing.SortOrder
+import javax.swing.table.TableRowSorter
+
+class CloudWatchLogsStreamsColumn : ColumnInfo<LogStream, String>(message("cloudwatch.logs.log_streams")) {
+    override fun valueOf(item: LogStream?): String? = item?.logStreamName()
+
+    override fun isCellEditable(item: LogStream?): Boolean = false
+}
+
+class CloudWatchLogsStreamsColumnDate : ColumnInfo<LogStream, String>(message("cloudwatch.logs.last_event_time")) {
+    override fun valueOf(item: LogStream?): String? {
+        val timestamp = item?.lastEventTimestamp() ?: return null
+        val date = DateFormatUtil.getDateFormat().format(timestamp)
```

Comment:
`DateFormatUtil.getDateTimeFormat` instead in case their locale has date/time in a separate order